core/arch/x86_code_test.c

```diff
@@ -142,6 +139,11 @@ unit_test_get_ymm_caller_saved()
             get_buffer[regno].u32[dword] = 0;
             ref_buffer[regno].u32[dword] = base++;
         }
+        for (int dword = sizeof(dr_ymm_t) / sizeof(uint);
+             dword < sizeof(dr_zmm_t) / sizeof(uint); ++dword) {
+            get_buffer[regno].u32[dword] = 0;
+            ref_buffer[regno].u32[dword] = 0;
```

Comment:
Why not just use memset instead of a custom loop?

Reply:
Sure, done.